kernel/src/process_manager/memory_channels.rs

```diff
@@ -0,0 +1,70 @@
+use crate::{address::VirtAddr, mm::PAGE_SIZE, process_manager::process_memory::ALLOCATION_RANGE_VIRT_START};
+
+use super::{allocation::AllocationRange, process::ProcessID, process_paging::ProcessPageTableRef};
+
+pub const INPUT_VADDR: u64 = 0xFF0000000000u64;
+pub const OUTPUT_VADDR: u64 = 0xFF8000000000u64;
```

Comment:
These exceed 48-bit virtual addresses. I suggest to use following instead
```suggestion
pub const INPUT_VADDR: u64 = 0x200_0000_0000u64;
pub const OUTPUT_VADDR: u64 = 0x210_0000_0000u64;
```

However, even with this change, I still cannot access input/output channels (it seems #PF)